src/lammpsio/box.py

```diff
@@ -4,22 +4,51 @@
 
 
 class Box:
-    """Triclinic simulation box.
+    r"""Simulation box.
 
-    The convention for defining the bounds of the box is based on
-    `LAMMPS <https://docs.lammps.org/Howto_triclinic.html>`_. This
-    means that the lower corner of the box is placed at ``low``, and
-    the size and shape of the box is determined by ``high`` and ``tilt``.
+    In LAMMPS, the simulation box is specified by three parameters: `low`,
+    `high`, and `tilt`. `low` defines the origin (lower corner) of the box,
+    while `high` specifies how far the box extends along each axis. The
+    difference between `high` and `low` gives three lengths $L_x$, $L_y$, and
+    $L_z$. `tilt` has three factors ($L_{xy}$, $L_{xz}$, $L_{yz}$) that skew the
+    edges to create non-orthorhombic simulation boxes. These parameters define
+    a box matrix consisting of three lattice vectors **a**, **b**, and **c**:
+
+    .. math::
+
+        [\mathbf{a} \quad \mathbf{b} \quad \mathbf{c} ] =
+        \begin{bmatrix}
+            L_x & L_{xy} & L_{xz} \\
+            0 & L_y & L_{yz} \\
+            0 & 0 & L_z
+        \end{bmatrix}
+
+    For more details on how to convert between the LAMMPS parameters and box
+    matrix see the `LAMMPS documentation
+    <https://docs.lammps.org/Howto_triclinic.html#transformation-from-general-to-restricted-triclinic-boxes>`_.
+
+    .. warning::
+
+        `high` is the upper bound of the simulation box **only** when it is
+        orthorhombic.
 
     Parameters
     ----------
     low : list
         Origin of the box
     high : list
-        "High" of the box, used to compute edge lengths.
+        High parameter used to compute edge lengths.
     tilt : list
-        Tilt factors ``xy``, ``xz``, and ``yz`` for a triclinic box.
-        Default of ``None`` is a strictly orthorhombic box.
+        Tilt factors ``xy``, ``xz``, and ``yz`` for a triclinic box. Default of
+        ``None`` is a strictly orthorhombic box, implying all are zero.
+
+    Examples
+    --------
+    Construct a triclinic simulation box:
+
+    .. code-block:: python
+
+        box = lammpsio.Box([-5.0, -10.0, 0.0], [1.0, 10.0, 8.0], [1.0, -2.0, 0.5])
```

Comment:
Show an example of orthorhombic in addition to triclinic, and explain what coordinates / size it's actually covering. This example doesn't tell us much right now. I also recommend using keywords for arguments to make the examples easier to read, throughout.